config/helm/aws-node-termination-handler/values.yaml

```diff
@@ -4,123 +4,129 @@
-enableRebalanceDraining: false
+# instanceMetadataURL is used to override the default metadata URL (default: http://169.254.169.254:80)
+instanceMetadataURL: ""
 
-# queueURL Listens for messages on the specified SQS queue URL
-queueURL: ""
+# Total number of times to try making the metadata request before failing.
+metadataTries: 3
 
 # checkASGTagBeforeDraining  If true, check that the instance is tagged with "aws-node-termination-handler/managed" as the key before draining the node
 checkASGTagBeforeDraining: true
 
 # managedAsgTag  The tag to ensure is on a node if checkASGTagBeforeDraining is true
 managedAsgTag: "aws-node-termination-handler/managed"
 
-# awsRegion If specified, use the AWS region for AWS API calls
-awsRegion: ""
-
-# awsEndpoint If specified, use the AWS endpoint to make API calls.
-awsEndpoint: ""
-
-# These should only be used for testing w/ localstack!
-awsSecretAccessKey:
-awsAccessKeyID:
-
-# enableSpotInterruptionDraining If false, do not drain nodes when the spot interruption termination notice is received
-enableSpotInterruptionDraining: ""
-
-# enableScheduledEventDraining [EXPERIMENTAL] If true, drain nodes before the maintenance window starts for an EC2 instance scheduled event
-enableScheduledEventDraining: ""
-
-# Total number of times to try making the metadata request before failing.
-metadataTries: 3
+# dryRun tells node-termination-handler to only log calls to kubernetes control plane
+dryRun: false
 
 # Cordon but do not drain nodes upon spot interruption termination notice.
 cordonOnly: false
 
 # Taint node upon spot interruption termination notice.
 taintNode: false
 
-# Log messages in JSON format.
-jsonLogging: false
-
-# Sets the log level
-logLevel: "info"
-
-# dryRun tells node-termination-handler to only log calls to kubernetes control plane
-dryRun: false
-
 # deleteLocalData tells kubectl to continue even if there are pods using
 # emptyDir (local data that will be deleted when the node is drained).
-deleteLocalData: ""
+deleteLocalData: true
 
 # ignoreDaemonSets causes kubectl to skip Daemon Set managed pods.
-ignoreDaemonSets: ""
+ignoreDaemonSets: true
 
-# gracePeriod (DEPRECATED - use podTerminationGracePeriod instead) is time in seconds given to each pod to terminate gracefully.
-# If negative, the default value specified in the pod will be used.
-gracePeriod: ""
-podTerminationGracePeriod: ""
+# podTerminationGracePeriod is time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used.
+podTerminationGracePeriod: -1
 
 # nodeTerminationGracePeriod specifies the period of time in seconds given to each NODE to terminate gracefully. Node draining will be scheduled based on this value to optimize the amount of compute time, but still safely drain the node before an event.
-nodeTerminationGracePeriod: ""
+nodeTerminationGracePeriod: 120
```

Comment:
If we were doing a bigger re-work (see above), I'd suggest removing the named variables from the chart and having people set ENV variables directly. This would stop chart drift and keep the chart as simple as possible. there could still be examples.

Reply:
So we're going to leave this for v2 then?